src/hooks/useUsersAttempts.js

```diff
@@ -244,10 +233,8 @@ const useUsersAttempts = ({ wordLength, correctWord, letters, setLoading }) => {
             photo,
           },
         };
-        const { id: dailyResultsId } = await addDoc(
-          collection(firebaseDb, DAILY_RESULTS),
-          newDailyResults
-        );
+
+        const dailyResultsId = await addDailyResults(newDailyResults);
```

Comment:
are you handling the exception at repository level or when gets called from the hook?

Reply:
I dont really have a nice error handling right now. I actually have a card for this 😅 I should add something like the ErrorBoundary at app level. Right now Im suppose to do it in the hook, but I don't have it in all cases. Sometimes I just log the error, and sometimes I don't even take it into account 😅  Here in this PR I actually just move the logic I had from the hook to another separate file.

<img width="325" alt="image" src="https://user-images.githubusercontent.com/8755889/217925604-278d6bd8-c625-4a08-85b7-94c554a2e2c2.png">

Reply:
haha sure! no problem!